src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure implements GetFigureArea {
+    private double firstLeg;
+    private double secondLeg;
+
+    public void draw() {
+        System.out.println("Figure - RightTriangle: firstLeg " + this.firstLeg
+                + " ,secondLeg " + this.secondLeg
+                + " ,area " + this.getArea() + " color "
+                + this.getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return  firstLeg * secondLeg / 2;
+    }
+    public RightTriangle(double firstLeg, double secondLeg) {

```

Comment:
The constructor for `RightTriangle` should also initialize the color if it is inherited from the `Figure` class. Consider adding a parameter for color and setting it using a setter method or directly if accessible.